packages/ai-core/src/artifact-api.test.ts

```diff
@@ -0,0 +1,76 @@
+import nock from 'nock';
+import { HttpDestination } from '@sap-cloud-sdk/connectivity';
+import {
+  ArtifactApi,
+  ArtifactCreationResponse,
+  ArtifactList,
+  ArtifactPostData
+} from './index.js';
+
+describe('artifact', () => {
+  const destination: HttpDestination = {
+    url: 'https://ai.example.com'
+  };
+
+  afterEach(() => {
+    nock.cleanAll();
+  });
+
+  it('get artifact parses a successful response', async () => {
```

Comment:
[req] `It` should read like a sentence and describe how the test behaves.  E.g., `parses a successful response for get request`